proto/internalapi/scanner/v4/vulnerability_report.proto

```diff
@@ -53,6 +53,13 @@ message VulnerabilityReport {
     string fixed_in_version = 11;
     CVSS cvss = 12 [deprecated = true]; // value of cvss field is included in cvss_metrics field, the exact deprecation date is undecided
     repeated CVSS cvss_metrics = 13;
+    message EPSS {
+      string date = 1;
+      string model_version = 2;
+      float probability = 3;
+      float percentile = 4;
```

Comment:
why store date and model version?

Reply:
Date: Since EPSS data is refreshed daily, including the date could be useful for reference.
Model Version: Users might want to know which model was used to generate the scores.
I think it’s better to include all relevant details in the vulnerability report, but we don't necessarily need to convert everything if some are not required.